.changeset/evil-times-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@api3/contracts': minor
+---
+
+Add dAPI and Market support to injective-evm-testnet and katana-sepolia-testnet
```

Comment:
We can no longer add dAPI support without Market support (https://github.com/api3dao/contracts/blob/main/data/chain-support.json) so this should be "Add Market support..."